internal/tsoptions/tsconfigparsing_test.go

```diff
@@ -706,7 +721,7 @@ func TestParseSrcCompiler(t *testing.T) {
 
 	opts := parseConfigFileContent.CompilerOptions()
 	assert.DeepEqual(t, opts, &core.CompilerOptions{
-		Lib:                        []string{"es2020"},
+		Lib:                        []string{"lib.es2020.d.ts"},
```

Comment:
This doesn't seem right; that's not what's in the config file and I don't think what Strada parses out.

Reply:
I think `tsoptions.GetLibFileName` has already handled this, and both "es2020" and "lib.es2020.d.ts" are valid for the function.
But I wonder which one the parser should use to set as the value for an enum option? It uses the value of map for top level enum option like `target` and uses the key for elements of a list instead. 🤔

Reply:
I think Strada returns the value of map in `convertJsonOptionOfCustomType`

Reply:
Whatever the Strada code did, this code should do.

Reply:
I dumped the options from Strada's [`parseOwnConfigOfJsonSourceFile`](https://github.com/microsoft/TypeScript/blob/release-5.8/src/compiler/commandLineParser.ts#L3553) and have confirmed that Strada used the value as option value:

![ts](https://github.com/user-attachments/assets/65f48fd5-f161-4032-b952-c08ec03691f7)